app/src/main/java/com/example/puzzlepic/ui/main/MainFragment.kt

```diff
@@ -42,16 +42,12 @@ class MainFragment : Fragment() {
     private lateinit var currentPhotoPath: String
     lateinit var navController: NavController
 
-    protected var puzzleURI : Uri? = null
+    protected var puzzleUri : Uri? = null
```

Comment:
Coding conventions suggested acronyms follow camelCase unless the acronym is only two letters
puzzleURI --->  puzzleUri
randomURL ---> randomUrl

Reply:
The jury is still out on this one. Generally, either approach is acceptable when using an acronym.  Do you have a source to cite for your choice?